gapic/src/main/com/google/gapid/models/Geometries.java

```diff
@@ -141,8 +141,7 @@ protected boolean isSourceComplete(DeviceDependentModel.Source<Source> source) {
       return success(result.get());
     } catch (DataUnavailableException e) {
       DeviceDependentModel.Source<Source> s = getSource();
-      // TODO: don't assume that it's because of not selecting a draw call.
-      return success(new Data(s.device, s.source.semantics, null, null));
+      return error(Loadable.Message.error(e));
```

Comment:
`DataUnavailableException` shouldn't be treated as an error. We treat it differently in the UI everywhere else. It should not show the exclamation point. Also, is the message from the server here actually helpful?